README.md

```diff
@@ -1,61 +1,27 @@
 React Firebase
 ==============
 
-React bindings for [Firebase](https://www.firebase.com).
+React bindings for [Firebase](https://firebase.google.com).
 
 ## Installation
 
-React Firebase requires **React 0.14 or later.**
+React Firebase requires **React 0.14 and Firebase 3 or later.**
 
 ```
 npm install --save react-firebase
 ```
 
 ## Usage
```

Comment:
Would also suggest adding a section for `# Examples` which describes the examples and how to run them.